include/fi_indexer.h

```diff
@@ -58,9 +58,10 @@ union ofi_idx_entry {
 
 struct indexer
 {
```

Comment:
What about to move this bracket to the previous line?

Reply:
Sure, but that's a minor code style issue which can be fixed in another PR. I don't to want to restart the CI by fixing it as the Mac OS builds are taking forever to complete. :)